manager/dm-server/src/main/java/org/apache/doris/stack/agent/AgentHeatbeatRunner.java

```diff
@@ -76,7 +76,7 @@ private void heartbeatCheck() {
                 agent.setStatus(AgentStatus.STOP);
                 agentRepository.save(agent);
                 agentCache.putAgent(agent);
-                log.warn("agent {} is unhealthly", agent.getHost());
+                log.warn("agent {} is unhealthily", agent.getHost());
```

Comment:
```suggestion
                log.warn("agent {} is unhealthy", agent.getHost());
```

Reply:
ok，thank you

Reply:
already modify "unhealthily" to "unhealthy"